src/api/app/models/package_issue.rb

```diff
@@ -25,7 +25,9 @@ def self.sync_relations(package, issues)
 
         # set change value for all
         issues.each do |pair|
-          PackageIssue.where(package: package, issue: pair.last).lock(true).update_all(change: pair.first)
+          PackageIssue.where(package: package, issue: pair.last).lock(true).find_each do |package_issue|
+            package_issue.update(change: pair.first)
```

Comment:
yes, but then you depend on the database, and the app should ensure that it has valid data independently of the database. But it is true that there are no validations for this model and it only has this method, so I guess we can avoid the validations in this case. :thinking: